### PR TITLE
Traduction française

### PR DESCRIPTION
**Pull request by Brainc3ll**
Greetings from Charleroi - Belgium !  Here is my contribution to this awesome addon.

I haven't had any LUA errors yet, but I haven't had to test every situation yet

### CODE REVIEW

File: Locales/frFR.lua [hunk 1]
```diff
@@ -1,6 +1,133 @@
+--Reserved space below so all localization files line up
 
-if not (GetLocale() == "frFR") then return end;
```
Comment (Peterodox) [inline Locales/frFR.lua:2]: Keep this so it doesn't overwrite the locales on non-french clients.

**Comment by Peterodox** [comment]
Hello! Thank you so much for providing the translation. How did you even find this unannounced project :joy:

File: Locales/frFR.lua (hunk 1 above)
Comment (Brainc3ll) [reply Locales/frFR.lua:2]: Oh, sorry, i'm not familiar with LUA, but i keep learning.

**Comment by Brainc3ll** [comment]
I always look at github repo of anddons and authors i like, to see if i can provide translation ;)

**Pull request merged**